doc/source/getting_started/intro_tutorials/08_combine_dataframes.rst

```diff
@@ -305,7 +305,7 @@ More information on join/merge of tables is provided in the user guide section o
     <div class="shadow gs-callout gs-callout-remember">
         <h4>REMEMBER</h4>
 
--  Multiple tables can be concatenated both column as row wise using
+-  Multiple tables can be concatenated both column- and row-wise using
```

Comment:
Maybe `column-wise and row-wise`

Reply:
There was a similar wording in `getting_started/basics`, as `...row- or column-wise, or elementwise.`, I tried to be consistent. Your suggestion also seems suitable.